File: java/pr9/src/main/java/com/company/pr9/entity/Station.java
package com.company.pr9.entity;

import io.jmix.core.entity.annotation.JmixGeneratedValue;
import io.jmix.core.metamodel.annotation.InstanceName;
import io.jmix.core.metamodel.annotation.JmixEntity;

import javax.persistence.*;
import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.Id;
import javax.persistence.Table;
import java.util.UUID;
import java.util.List;

@JmixEntity
@Table(name = "TR_STATION")
@Entity(name = "tr_Station")
public class Station {
    @JmixGeneratedValue
    @Column(name = "ID", nullable = false)
    @Id
    private UUID id;

    public UUID getId() {
        return id;
    }

    public void setId(UUID id) {
        this.id = id;
    }

    @InstanceName
    @Column(name = "NAME")
    private String name;

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    @OneToMany(mappedBy = "station")
    private List<Route> routes;

    public List<Route> getRoutes() {
        return routes;
    }

    public void setRoutes(List<Route> routes) {
        this.routes = routes;
    }

    @OneToMany(mappedBy = "station")
    private List<Bus> buses;

    public List<Bus> getBuses() {
        return buses;
    }

    public void setBuses(List<Bus> buses) {
        this.buses = buses;
    }

}

File: java/pr9/src/main/kotlin/com/company/pr9/entity/Station.kt
package com.company.pr9.entity

import io.jmix.core.entity.annotation.JmixGeneratedValue
import io.jmix.core.metamodel.annotation.InstanceName
import io.jmix.core.metamodel.annotation.JmixEntity
import java.util.*
import javax.persistence.*

@JmixEntity
@Table(name = "STATION")
@Entity
open class Station {
    @JmixGeneratedValue
    @Column(name = "ID", nullable = false)
    @Id
    private val id: UUID? = null

    @InstanceName
    @Column(name = "NAME")
    private val name: String? = null

    @OneToMany(mappedBy = "station")
    private val routes: List<Route> = ArrayList<Route>()

    @OneToMany(mappedBy = "station")
    private val buses: List<Bus> = ArrayList() // Constructors, getters, setters
}
